examples/jxlinfo.c

```diff
@@ -25,9 +25,9 @@ int PrintBasicInfo(FILE* file) {
 
   JxlDecoderSetKeepOrientation(dec, 1);
 
-  if (JXL_DEC_SUCCESS !=
-      JxlDecoderSubscribeEvents(
-          dec, JXL_DEC_BASIC_INFO | JXL_DEC_COLOR_ENCODING | JXL_DEC_FRAME)) {
+  if (JXL_DEC_SUCCESS != JxlDecoderSubscribeEvents(
+                             dec, JXL_DEC_BASIC_INFO | JXL_DEC_COLOR_ENCODING |
+                                      JXL_DEC_FRAME | JXL_DEC_BOX)) {
```

Comment:
I find the indentation odd here

Reply:
I agree with that, but unfortunately ./ci.sh lint | patch -p1 keeps producing this indentation even when starting with the whole statement on one line

I prefer to keep the automated output anyway

Reply:
Agreed, very peculiar output from clang-format...